### PR TITLE
Dyn 6821 home page links fix

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

Addresses this Jira issue: [DYN-6821](https://jira.autodesk.com/browse/DYN-6821) Fixes HomePage links.

image

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [x] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB

### Release Notes

- fixes homepage links
- this was broken during the drag&drop implementation

### Reviewers

@reddyashish 

### FYIs

@Amoursol

### CODE REVIEW

**Comment by QilongTang** [comment]
Sorry @dnenov I think our team rebased master branch so your `DYN-6821-HomePage-links-fix` had the wrong base branch and contained changes unintended for master now, would you rebase your changes?

**Comment by QilongTang** [comment]
hi @dnenov I think you still have a few deletion change needs to be reverted

File: src/Libraries/DSOffice/Properties/Resources.resx [hunk 1]
```diff
@@ -1,123 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (reddyashish) [inline src/Libraries/DSOffice/Properties/Resources.resx:1]: Are these resources not used anymore?

Comment (dnenov) [reply src/Libraries/DSOffice/Properties/Resources.resx]: Yes, I missed those .. I'm sorry guys.

**Review by QilongTang — APPROVED** [review]

**Pull request merged**